src/party_events.ts

```diff
@@ -76,3 +76,51 @@ export const getAllPartyEvents = async (fromBlock: number) => {
 
   return events;
 };
+
+export const haveSetNewPartyWithContribution = async (
+  partyAddress: string
+): Promise<boolean> => {
+  const key = `new_party_with_contribution_${partyAddress}`;
+  const res = await getRedisAsync(key);
+  if (res) {
+    console.info(
+      `already alerted about new party with contribution ${partyAddress}`
+    );
+    return true;
+  } else {
+    console.info(
+      `have NOT alerted about new party with contribution ${partyAddress}`
+    );
+    return false;
+  }
+};
+
+export const setNewPartyWithContribution = async (partyAddress: string) => {
+  const key = `new_party_with_contribution_${partyAddress}`;
```

Comment:
can we move the creation of this key to a helper fn? we use it here and in fn above